rviz_mesh_tools_plugins/src/MapDisplay.cpp

```diff
@@ -731,11 +830,12 @@ void MapDisplay::saveLabel(Cluster cluster)
       return;
     }
 
-    // Open IO
-    hdf5_map_io::HDF5MapIO map_io(m_mapFilePath->getFilename());
+    // is not working anyway
+    // // Open IO
+    // hdf5_map_io::HDF5MapIO map_io(m_mapFilePath->getFilename());
 
-    // Add label with faces list
-    map_io.addOrUpdateLabel(results[0], results[1], faces);
+    // // Add label with faces list
+    // map_io.addOrUpdateLabel(results[0], results[1], faces);
```

Comment:
I'd be fine with fully removing these lines of code. I usually prefer having less clutter over seeing how things were done in the past. If we implement e.g. color/texture loading later and some of the commented out code is doing something non-trivial, we can always look at the ROS1 version in git, for example.

The same applies for the two larger comment blocks above in this file.

Reply:
I removed all the larger comment blocks